Kernel/Applications/spinner.cpp

```diff
@@ -10,19 +10,37 @@
  */
 #include <Applications/spinner.hpp>
 #include <Devices/Graphics/console.hpp>
+#include <Memory/Heap.hpp>
+#include <Library/string.hpp>
+#include <Arch/Memory.hpp>
+#include <Panic.hpp>
 
 namespace Apps {
 
 void spinner(void) {
     Console::printf("\n");
     int i = 0;
     const char spinnay[] = { '|', '/', '-', '\\' };
+    for (size_t size = 1; size < 1024 * 1024 * 1024; size++) {
```

Comment:
I'm offended that you didn't just write a new task for this test. Poor `spinner` doesn't deserve to be abused like this.

Reply:
It might also be nice to test this code outside of the kernel so that it's easier to debug.

Reply:
Reset the spinner in 8230d4fec34dadabde6c8370b30c4787170b2129. Its suffering is over.